### PR TITLE
Add some documentation for container structure tests

### LINKED ISSUE #1917: Improve documentation on testing phase
https://skaffold.dev/docs/how-tos/testers has no documentation, we should add at least a brief explanation on how skaffold runs tests and the what tests it supports.

ref #1076

### PR DESCRIPTION
**Pull request by corneliusweig**
Close #1917

### CODE REVIEW

**Comment by corneliusweig** [comment]
@nkubala you are predestined to look at this :wink:

File: docs/content/en/docs/how-tos/testers/_index.md [hunk 1]
```diff
@@ -6,4 +6,25 @@ weight: 15
 
 This page discusses how to set up Skaffold to run container structure tests after building an artifact.
 
-{{% todo 1076 %}}
+Container structure tests are consistency checks for containers.
+Skaffold relies on [container-structure-test](https://github.com/GoogleContainerTools/container-structure-test) to execute those tests, and requires its [binary](https://github.com/GoogleContainerTools/container-structure-test/releases) to be installed.
+
+When configured for a container image, the structure tests are run after every container build.
```
Comment (priyawadhwa) [inline docs/content/en/docs/how-tos/testers/_index.md]: This line is a bit confusing, maybe

Container structure tests are defined per image in the skaffold config. Every time an artifact is rebuilt, skaffold will run the associated structure tests on that image. If the tests fail, skaffold will not continue on to the deploy stage.

WDYT?

File: docs/content/en/docs/how-tos/testers/_index.md [hunk 2]
```diff
@@ -6,4 +6,25 @@ weight: 15
...
+If this is prohibitive, long-running tests should be moved to a different Skaffold profile.
+
+### Example
+The following `test` section, instructs Skaffold to run all container structure tests in the `structure-test` folder relative to the Skaffold root directory:
```
Comment (priyawadhwa) [inline docs/content/en/docs/how-tos/testers/_index.md]: I would mention that this section is in the skaffold.yaml just for clarity

File: docs/content/en/docs/how-tos/testers/_index.md (hunk 1 above)
Comment (corneliusweig) [reply docs/content/en/docs/how-tos/testers/_index.md]: That's perfect. I totally missed that failing tests would stop the deployment. Good catch!

**Comment by corneliusweig** [comment]
@priyawadhwa Thanks for your enhancements! PTAL

**Pull request merged**